codecov_auth/models.py

```diff
@@ -71,8 +71,8 @@ class User(BaseCodecovModel):
     is_staff = models.BooleanField(null=True, default=False)
     is_superuser = models.BooleanField(null=True, default=False)
     external_id = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
-    terms_agreement = models.BooleanField(null=True, default=False)
-    terms_agreement_at = DateTimeWithoutTZField(null=True)
+    terms_agreement = models.BooleanField(null=True, default=None, blank=True)
```

Comment:
We had this signal, https://github.com/codecov/codecov-api/blob/main/codecov_auth/signals.py, when an owner was created to create an `OwnerProfile` with `terms_agreement` set to false by default. I don't think we want to set this to None here, rather to `False` as None and False are treated different by the UI

Reply:
Updated to default new users to have False value.